Card/src/main/java/com/braintreepayments/api/BaseCard.java

```diff
@@ -101,6 +102,7 @@ public void setExpirationDate(@Nullable String expirationDate) {
         if (TextUtils.isEmpty(expirationDate)) {
             expirationMonth = null;
             expirationYear = null;
+            this.expirationDate = null;
```

Comment:
We had a setExpirationDate method, but no getter so the property wasn't synthesized in Kotlin. There wasn't a member variable for expirationDate since it was directly being converted to month and year in the setter, so I added a variable to allow for a getter and the property in Kotlin. Open to other ideas on how to fix this property.

Reply:
Another option, instead of adding the member variable, would be to have the expiration date setter reconstruct the expiration date from the month and year member variables.

Reply:
Good call - we decided to remove the local variable and reconstruct it 👍